modules/nixos/monitoring/prometheus.nix

```diff
@@ -48,6 +41,11 @@
     ];
   };
 
+  services.telegraf.extraConfig.inputs.prometheus.urls = [
+    "http://localhost:9090/metrics" # prometheus
+    "http://localhost:9093/metrics" # alertmanager
```

Comment:
@Mic92 Do you have something like this in your config and I've overlooked it?

I don't think rules like `alerts_silences_changed`, `prometheus_notifications_alertmanagers_discovered` will work without it?

Reply:
Merging this PR but if you think this should be handled differently let me know and I'll change it.